planner/core/physical_plan_test.go

```diff
@@ -201,6 +202,14 @@ func (s *testPlanSuite) TestDAGPlanBuilderSimpleCase(c *C) {
 			sql:  "select * from ((SELECT 1 a,6 b) UNION (SELECT 2,5) UNION (SELECT 2, 4) ORDER BY 1) t order by 1, 2",
 			best: "UnionAll{Dual->Projection->Dual->Projection->Dual->Projection}->HashAgg->Sort->Sort",
 		},
+		{
+			sql:  "select * from (select *, NULL as xxx from t) t order by xxx",
```

Comment:
This test can work well without this commit.
Maybe we need to build another case?

Reply:
Let me check it later.